assets/css/_map_page.scss

```diff
@@ -85,6 +97,26 @@ $z-map-page-context: (
   }
 }
 
+.c-map-page__panel-backdrop-button {
+  display: none;
+
+  .c-map-page__input-and-results--visible ~ & {
+    @media screen and (max-width: $mobile-portrait-max-width) {
+      // Make button clickable
+      display: block;
+      // Force Flex + Posistion to fill area
+      width: 100%;
+
+      // Ignore flex layout and fill screen
+      position: absolute;
+      inset: 0;
+
+      // Inbetween panel and map
+      z-index: 150;
```

Comment:
**suggestion:** could you define this in terms of the values from `$z-map-page-context`?

Reply:
Thanks for the reminder!
12a3a0a2a54997d2e8a2ccba36eb0d39de5b7923